### PR TITLE
init frontend project

### PR DESCRIPTION
**Pull request by LeonTrestman**

### CODE REVIEW

File: frontend/src/pages/call/create.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { type NextPageWithAuth } from "next";
+
+const Create: NextPageWithAuth = () => {
+  return <div>call/create</div>;
+};
+
+Create.auth = { requiredRole: "user" };
```
Comment (maorbjfrog) [inline frontend/src/pages/call/create.tsx:7]: you mean customer?

File: frontend/src/pages/call/status.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { type NextPageWithAuth } from "next";
+
+const Status: NextPageWithAuth = () => {
+  return <div>call/status</div>;
+};
+
+Status.auth = { requiredRole: "user" };
```
Comment (maorbjfrog) [inline frontend/src/pages/call/status.tsx:7]: you mean customer?

File: frontend/src/pages/call/create.tsx (hunk 1 above)
Comment (LeonTrestman) [reply frontend/src/pages/call/create.tsx:7]: yes

File: frontend/src/pages/call/status.tsx (hunk 2 above)
Comment (LeonTrestman) [reply frontend/src/pages/call/status.tsx:7]: changed

**Pull request merged**